tst/testinstall/weakptr-badargs.tst

```diff
@@ -4,7 +4,7 @@ gap> START_TEST("weakptr-badargs.tst");
 gap> w := WeakPointerObject([1,,3,4]);
 Syntax warning: Unbound global variable in stream:1
 w := WeakPointerObject([1,,3,4]);
-                      ^
+     ^^^^^^^^^^^^^^^^^
 Error, Variable: 'WeakPointerObject' must have a value
```

Comment:
I know this is not really part of this PR, but... Why is this test even here?

Reply:
Dunno :)

Reply:
Well, I thought you might now, as you added it in 35267ae :-)

Reply:
If I had to guess, I'd suspect it's just a one line too long cut+paste of a manually written test :)